sdk/src/types/block/payload/signed_transaction/mod.rs

```diff
@@ -46,12 +45,9 @@ impl SignedTransactionPayload {
 
     /// Computes the identifier of a [`SignedTransactionPayload`].
     pub fn id(&self) -> TransactionId {
```

Comment:
ID is a property of the `Transaction`, I would put it there. I guess it's fine to keep this one and forward though